pepys_import/core/store/common_db.py

```diff
@@ -1354,29 +1354,17 @@ def sensor_name(self):
     def platform(self):
         return relationship(
             "Platform",
-            lazy="joined",
-            join_depth=1,
-            innerjoin=True,
-            uselist=False,
             foreign_keys="Media.platform_id",
         )
 
     @declared_attr
     def platform_name(self):
         return association_proxy("platform", "name")
 
-    @declared_attr
```

Comment:
Is there a reason this is removed? I wonder if, now we've got the platform relationship back in, can we add this back in too?